frontend/src/Tests/TestExpandibleTable.jsx

```diff
@@ -0,0 +1,221 @@
+import ExpandableTable from "../components/Common/ExpandableTable";
```

Comment:
The component is called ExpandableTable, I think this test file has a typo because is called TestExpandibleTable ( with "i" instead of a). Both expandable and expandible are valid but for consistency we can stick with one for all the files.

Reply:
I changed it to expandable!